kolibri/core/apps.py

```diff
@@ -84,3 +88,51 @@ def activate_pragmas_on_start():
             # at the cost of a slight penalty to all reads.
             cursor.execute(START_PRAGMAS)
             connection.close()
+
+    # flake8: noqa: C901
+    @staticmethod
+    def check_redis_settings():
+        """
+        Check that Redis settings are sensible, and use the lower level Redis client to make updates
+        if we are configured to do so, and if we should, otherwise make some logging noise.
+        """
+        if OPTIONS["Cache"]["CACHE_BACKEND"] != "redis":
+            return
+        config_use_conf = OPTIONS["Cache"]["CACHE_REDIS_USE_CONF"]
+        config_maxmemory = OPTIONS["Cache"]["CACHE_REDIS_MAXMEMORY"]
+        config_maxmemory_policy = OPTIONS["Cache"]["CACHE_REDIS_MAXMEMORY_POLICY"]
+
+        try:
+            # see redis_cache.backends.single.RedisCache
+            helper = RedisSettingsHelper(process_cache.get_master_client())
+
+            # default setting is "0", or no limit
+            maxmemory = helper.get_maxmemory()
+            if config_use_conf:
+                if maxmemory == 0:
+                    logger.warning("Redis is configured without a maximum memory size.")
+            elif maxmemory != config_maxmemory:
```

Comment:
I don't think this is a good idea: to change the `maxmemory` parameter of redis, the user should have requested it explictly.
The fact that the user wants to use redis as Cache does not justify changing the maxmemory value if the user does not set it.
The user might be using redis for other purposes and Kolibri should not change its configuration without the explicit config parameter. 
I think the above if should replace `config_use_conf` by `config_maxmemory`